src/components/RightSidebar/Participants/ParticipantsTab.vue

```diff
@@ -12,9 +12,17 @@
 				:value.sync="searchText"
 				:is-focused.sync="isFocused"
 				:placeholder-text="searchBoxPlaceholder"
+				:aria-describedby="showSearchBoxDescription && searchBoxDescriptionId"
 				@input="handleInput"
 				@keydown.enter="addParticipants(participantPhoneItem)"
 				@abort-search="abortSearch" />
+			<div v-if="showSearchBoxDescription"
+				:id="searchBoxDescriptionId"
+				:title="searchBoxDescription"
+				class="search-form__description">
+				<IconInformationOutline :size="20" />
+				<span class="hidden-visually">{{ searchBoxDescription }}</span>
```

Comment:
Wouldn't aria-label be sufficient here? If not, I'm afraid, we might accidentally delete it in future, if it won't be specifically commented

Reply:
Alternative is to remove the text here and set it as `aria-desription` on the input. But then I'm afraid it's even simpler to lose, because it would have no direct relation with the element.